src/app/dim-ui/FractionalPowerLevel.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import styles from './FractionalPowerLevel.m.scss';
+
+export function FractionalPowerLevel({ power }: { power: number }) {
+  const numerator = (power * 8) % 8 || null;
+  return (
+    <span className={styles.fractionalPowerLevel}>
+      {Math.floor(power)}
+      {numerator && (
+        <>
+          &#8239;
+          <span className={styles.fraction}>
+            <sup>{numerator}</sup>⁄<sub>8</sub>
```

Comment:
I assume you specifically didn't want to use the ⅝ fraction glyphs because there's no 2/8?

Reply:
they're generally smaller, break line height if you enlarge them, less visually fine tunable, and required a lookup table